pypowsybl/voltage_initializer.py

```diff
@@ -0,0 +1,137 @@
+from typing import Dict, List, Tuple
+from pypowsybl import _pypowsybl as _pp
+from pypowsybl.network import Network as _Network
+
+
+class VoltageInitializerParameters:
+    """
+    """
```

Comment:
can you add a main comment on what does these class

Reply:
Done